### DISCUSSION TITLE
help me, please!!!!

### DISCUSSION THREAD

**Discussion by congviecnewnew** [community/community#35727] [Other Feature Feedback, Questions, & Ideas]
Dear, 
i get your project  from https://github.com/wblachowski/camculator. 
when i  use load math  
private fun loadMath() {
        setStatus(R.string.loading_math)
        try {
           F.await()

question 1:  when run app on android , screen will display loading_math text. after app auto quit, exit. when i delete  F.await() line, app run normal. is error  F.await()? as picture 1. how to fix?. 
Question 2: after delete  F.await(), app run.  i capture a photo as picture 2,3,4. equation is right, but it can not solution.
help me, please!
Thanks so much for your help!

**Comment by airtower-luna** [comment]
It's best if you look at support options listed in the repository you're asking about, if there's nothing listed try creating an issue. That way it's much more likely that the developer(s) will see your question. :wink:
